kexec.c

```diff
@@ -65,6 +65,8 @@ struct kexec_segment {
 
 #define	LINUX_REBOOT_CMD_KEXEC	0x45584543
 
+#define __NR_kexec_file_load	382
```

Comment:
What about something like 
```
#if !defined(__NR_kexec_file_load)
# define __NR_kexec_file_load	382
#endif
```
Then this will work with newer header files that define it..